openjdk/playlist.xml

```diff
@@ -86,7 +86,10 @@
 		<levels>
 			<level>extended</level>
 		</levels>
-		<!-- temporarily disable on openj9 due to https://github.com/eclipse/openj9/issues/10757 -->
+        <disabled>
```

Comment:
Thanks @jwnmoon - can you fix the indentation and move the disabled block up to just below `<testCaseName>` like in the style of https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/perf/renaissance/playlist.xml#L30-L35 (for this and all cases you have modified in this file?

Reply:
```
		<disabled>
                        <comment>https://github.com/AdoptOpenJDK/openjdk-tests/issues/2157#issuecomment- 
                        758317252</comment>
                        <comment>https://github.com/eclipse/openj9/issues/10757</comment>
                        <impl>openj9</impl>
		</disabled>
```
Thank you so much for the review! 
I'm working on revising the code, and I came across some test cases that ended up with 2 comments. Is this the correct way to show multiple comments?

Reply:
For this particular test `jfc_demo` you would put 2 disabled blocks

```
<disabled>
        <comment>https://github.com/AdoptOpenJDK/openjdk-tests/issues/2157#issuecomment-758317252</comment>
        <impl>hotspot</impl>
</disabled>
<disabled>
        <comment>https://github.com/eclipse/openj9/issues/10757</comment>
        <impl>openj9</impl>
</disabled>
```
similar to the 2nd example in the https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/userGuide.md#exclude-test-against-multiple-criteria section of the User Documentation.

Reply:
I pushed the document after revising everything, but for some reason the indentation is off when I see it on Github. Everything looks fine in the document itself. Is there a way to fix this?
<img width="656" alt="Screen Shot 2021-03-31 at 7 52 19 PM" src="https://user-images.githubusercontent.com/73265390/113228486-9afe4880-925a-11eb-91cd-9f403128f1d7.png">
<img width="788" alt="Screen Shot 2021-03-31 at 7 52 27 PM" src="https://user-images.githubusercontent.com/73265390/113228495-9fc2fc80-925a-11eb-9eba-bde937a1cea2.png">